.storybook/manager-head.html

```diff
@@ -1,3 +1,9 @@
 <script>
   document.title = 'Canvas Kit Storybook';
 </script>
+
+<style>
+  [title^='_hidden_'] {
+    display: none !important;
+  }
+</style>
```

Comment:
This style rule will hide any Storybook item with a name that starts with "_hidden_". The `testingOnly` utility function will apply this.